app/controllers/ui_job_wizard_controller.rb

```diff
@@ -0,0 +1,14 @@
+class UiJobWizardController < ::Api::V2::BaseController
+  def categories
+    job_categories = resource_scope.where(['job_category LIKE ?', "%#{params[:search]}%"]).pluck(:job_category).uniq
```

Comment:
`JobTemplate` allows searching by `job_category` using scoped search so you should be able to do `resource_scope.search_for("job_category ~ #{params[:search]}")`

Also since you're after a list of unique job categories, you can let the db calculate it without having to load all the job categories and then removing the duplicates in ruby.
```ruby
resource_scope
  .where(['job_category LIKE ?', "%#{params[:search]}%"])
  .select(:job_category).distinct
  .reorder(:job_category)
  .pluck(:job_category)
```

Reply:
So should I combine the two 
``` ruby
resource_scope
  .search_for("job_category ~ #{params[:search]}")
  .select(:job_category).distinct
  .reorder(:job_category)
  .pluck(:job_category)
```
or just use the last one? 
``` ruby
resource_scope
  .where(['job_category LIKE ?', "%#{params[:search]}%"])
  .select(:job_category).distinct
  .reorder(:job_category)
  .pluck(:job_category)
```

Reply:
Combination would be nice

Reply:
Done!